### DISCUSSION TITLE
Index - Easier Access to Bookmarklets

### DISCUSSION THREAD

**Discussion by MajesticWafer** [dragon731012/-WORKING-bookmarklets-and-games#43] [Ideas]
Create an index for easier access to bookmarklets.

I have one in progress, and am developing it now. Tell me if you want anything specific for it, and if you want to also make it.

**Comment by MajesticWafer** [comment]
[Here’s](https://github.com/dragon731012/-WORKING-bookmarklets-and-games/files/15248908/index.txt) the index if you would like to use it.

**Reply by Wheels522** [reply]
Join da discord https://discord.gg/24kSEWa2
